crates/exex/src/atlantic/endpoints.rs

```diff
@@ -0,0 +1,266 @@
+use super::error::SharpSdkError;
+use url::Url;
+
+/// The L2 endpoints.
+#[derive(Debug, Clone)]
+pub struct L2Endpoints {
+    /// Submit an Atlantic proof generation query
+    pub proof_generation: Url,
+    /// Submit an Atlantic trace generation query
+    pub trace_generation: Url,
+    /// Submit an Atlantic trace generation and proof generation query
+    pub trace_generation_proof_generation: Url,
+    /// Submit a L2 Atlantic query
+    pub atlantic_query: Url,
+    /// Submit a L2 Atlantic proof generation and proof verification query
+    pub proof_generation_verification: Url,
+    /// Submit a L2 Atlantic proof verification query
+    pub proof_verification: Url,
+}
+
+impl L2Endpoints {
+    pub fn new(base_url: &Url) -> Result<Self, SharpSdkError> {
+        Ok(Self {
+            proof_generation: base_url.join("proof-generation")?,
+            trace_generation: base_url.join("trace-generation")?,
+            trace_generation_proof_generation: base_url
+                .join("trace-generation-proof-generation")?,
+            atlantic_query: base_url.join("l2/atlantic-query")?,
+            proof_generation_verification: base_url
+                .join("l2/atlantic-query/proof-generation-verification")?,
+            proof_verification: base_url.join("l2/atlantic-query/proof-verification")?,
+        })
+    }
+}
+
+/// The Sharp queries endpoints.
+#[derive(Debug, Clone)]
+pub struct SharpQueriesEndpoints {
+    /// Get the list of Atlantic queries submitted by the user
+    pub get_queries: Url,
+    /// Get the details of a specific Atlantic query
+    pub get_query: Url,
+    /// Get the list of jobs for a specific Atlantic query
+    pub get_query_jobs: Url,
+}
```

Comment:
also an enum here. And I think that the `get_` prefix is superfluous

Reply:
- Same comment for the enum
- Just fixed for the get prefixes